resources/js/components/shared/EllipsisMenu.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <b-dropdown
+    variant="ellipsis"
+    no-caret
+    no-flip
+    lazy
+    class="dropdown-right ellipsis-dropdown-main"
+  >
+    <template v-if="customButton" #button-content>
+      <i
+        class="pr-1 ellipsis-menu-icon"
+        :class="customButton.icon"
+      />
+      <span>{{ customButton.content }}</span>
+    </template>
+    <template v-else #button-content>
+      <i class="fas fa-ellipsis-h ellipsis-menu-icon" />
+    </template>
+    <div v-if="divider === true">
+      <b-dropdown-item
+        v-for="action in filterAboveDivider"
+        :key="action.value"
+        class="ellipsis-dropdown-item mx-auto"
+        @click="onClick(action, data)"
+      >
+        <div class="ellipsis-dropdown-content">
+          <i
+            class="pr-1"
+            :class="action.icon"
+          />
+          <span>{{ action.content }}</span>
+        </div>
+      </b-dropdown-item>
+      <b-dropdown-divider />
+      <b-dropdown-item
+        v-for="action in filterBelowDivider"
+        :key="action.value"
+        class="ellipsis-dropdown-item mx-auto"
+        @click="onClick(action, data)"
+      >
+        <div class="ellipsis-dropdown-content">
+          <i
+            class="pr-1"
+            :class="action.icon"
+          />
+          <span>{{ action.content }}</span>
+        </div>
+      </b-dropdown-item>
+    </div>
+    <div v-else>
+      <b-dropdown-item
+        v-for="action in filterActions"
+        :key="action.value"
+        class="ellipsis-dropdown-item mx-auto"
+        @click="onClick(action, data)"
+      >
+        <div class="ellipsis-dropdown-content">
+          <i
+            class="pr-1"
```

Comment:
I would consider adding `fa-fw` to the class of the icon here. [This will make it fixed-width](https://fontawesome.com/v5/docs/web/style/fixed-width) which is desirable in a list view so that the alignment of the text next to the icons is consistent.

Reply:
This has been updated!